src/core/expression/qgsexpressionfunction.cpp

```diff
@@ -4862,6 +4863,93 @@ bool QgsArrayForeachExpressionFunction::prepare( const QgsExpressionNodeFunction
   return true;
 }
 
+QgsArrayFilterExpressionFunction::QgsArrayFilterExpressionFunction()
+  : QgsExpressionFunction( QStringLiteral( "array_filter" ), 2, QCoreApplication::tr( "Arrays" ) )
```

Comment:
can you used named parameters here? I think with functions like this having the ability to name arguments helps with readability